<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemInfoWindow</class>
 <widget class="QMainWindow" name="SystemInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Information</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="tree_category">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true"/>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_save"/>
   <addaction name="action_print"/>
   <addaction name="action_refresh"/>
  </widget>
  <action name="action_save">
   <property name="icon">
    <iconset resource="../resources/client.qrc">
     <normaloff>:/img/disk.png</normaloff>:/img/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="action_print">
   <property name="icon">
    <iconset resource="../resources/client.qrc">
     <normaloff>:/img/printer.png</normaloff>:/img/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print (Ctrl+P)</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+P</string>
   </property>
  </action>
  <action name="action_copy_row">
   <property name="text">
    <string>Copy Row</string>
   </property>
  </action>
  <action name="action_copy_name">
   <property name="text">
    <string>Copy Name</string>
   </property>
  </action>
  <action name="action_copy_value">
   <property name="text">
    <string>Copy Value</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="icon">
    <iconset resource="../resources/client.qrc">
     <normaloff>:/img/arrow-circle-double.png</normaloff>:/img/arrow-circle-double.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh (F5)</string>
   </property>
   <property name="shortcut">
    <string notr="true">F5</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/client.qrc"/>
 </resources>
 <connections/>
</ui>
